src/components/Personalization/constants/eventType.js

```diff
@@ -0,0 +1,15 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+export const DECISIONING_PROPOSITION_DISPLAY = "decisioning.propositionDisplay";
+export const DECISIONING_PROPOSITION_INTERACT =
```

Comment:
This constant naming describes the value more than what it is. What would happen if we changed the value, would we then need to change the name of the constant to match? That would defeat the whole purpose of the constant.

How about `DISPLAY_EVENT_TYPE`, and `INTERACT_EVENT_TYPE` ?